staging/src/k8s.io/apimachinery/pkg/runtime/error.go

```diff
@@ -154,19 +155,21 @@ func (e *strictDecodingError) Errors() []error {
 // IsStrictDecodingError returns true if the error indicates that the provided object
 // strictness violations.
 func IsStrictDecodingError(err error) bool {
+	var serr *strictDecodingError
```

Comment:
nit: you can let `IsStrictDecodingError` call `AsStrictDecodingError`

Reply:
thx, updated